code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -7453,6 +7453,44 @@ var/global/list/bomb_like_items = list(/obj/item/device/transfer_valve, /obj/ite
 	reagents.add_reagent(ZAMSPICYTOXIN, 5)
 	bitesize = 3
 
+/obj/item/weapon/reagent_containers/food/snacks/blethernoodlesoup
+	name = "Blether Noodle Soup"
+	desc = "A hearty grey noodle soup. Great for teaching growing greylings new words! Not to be confused with human alphabet soup."
+	icon_state = "blethernoodlesoup_open"
+	trash = /obj/item/weapon/reagent_containers/glass/soupcan
+	food_flags = FOOD_MEAT | FOOD_LIQUID
+	crumb_icon = "dribbles"
+	filling_color = "#FF9700"
+	valid_utensils = UTENSILE_FORK|UTENSILE_SPOON
+	wrapped = 0
+
+/obj/item/weapon/reagent_containers/food/snacks/blethernoodlesoup/New()
+	..()
+	reagents.add_reagent(NUTRIMENT, 6)
+	reagents.add_reagent(SACID, 10)
+	reagents.add_reagent(LOCUTOGEN, 5)
+	bitesize = 3
+
+/obj/item/weapon/reagent_containers/food/snacks/blethernoodlesoup/wrapped
+	name = "Blether Noodle Soup"
+	desc = "A hearty grey noodle soup. Great for teaching growing greylings new words! Not to be confused with human alphabet soup."
+	icon_state = "blethernoodlesoup_closed"
+	wrapped = 1
```

Comment:
prefer this (and the one at 7492) to be TRUE and FALSE instead of just 1 and 0

Reply:
Fixed